src/server/game/Entities/Player/Player.cpp

```diff
@@ -17847,7 +17847,7 @@ void Player::_LoadAuras(PreparedQueryResult result, uint32 timediff)
             }
 
             // negative effects should continue counting down after logout
-            if (remaintime != -1 && !spellInfo->IsPositive())
+            if (remaintime != -1 && ((!spellInfo->IsPositive() && spellInfo->Id != 15007) || spellInfo->HasAttribute(SPELL_ATTR4_FADES_WHILE_LOGGED_OUT))) // Resurrection sickness should not fade while logged out
```

Comment:
https://us.battle.net/forums/en/wow/topic/1670088555
https://us.battle.net/forums/en/wow/topic/2228173818
https://us.battle.net/forums/en/wow/topic/2369682005

so, @xinef1 is right